src/engagement_db_to_analysis/message_filters.py

```diff
@@ -0,0 +1,67 @@
+from core_data_modules.logging import Logger
+from dateutil.parser import isoparse
+import time
+
+from core_data_modules.traced_data import Metadata
+
+from src.pipeline_configuration_spec import *
+
+
+log = Logger(__name__)
+
+
+# TODO: Move to Core.
+class Filters(object):
```

Comment:
Given that these are all static functions, the more pythonic way to write this is to put the functions in a module, rather than wrapping in a class like this (I too was wrong about this for a long time, as in other languages this approach would be normal).

Reply:
refactored to `message_filters` module.